local-pv-provisioner/cmd/root.go

```diff
@@ -0,0 +1,66 @@
+package cmd
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"k8s.io/klog"
+)
+
+var config struct {
+	metricsAddr      string
+	nodeName         string
+	deviceDir        string
+	deviceNameFilter string
+	development      bool
+	pollingInterval  int
```

Comment:
should be a `time.Duration` as suggested below.

Reply:
Changed to use Duration.